app-mock/src/androidTest/java/pl/dybczak/githubusers/utils/noViewMatcher.kt

```diff
@@ -0,0 +1,21 @@
+package pl.dybczak.githubusers.utils
+
+import android.view.View
+import androidx.test.espresso.Espresso
+import androidx.test.espresso.NoMatchingViewException
+import androidx.test.espresso.action.ViewActions
+import org.hamcrest.Matcher
+
+object noViewMatcher {
```

Comment:
It contains multiple matchers, so maybe more generic name could be a better choice here? (please remember to update file name as well).
```suggestion
object ViewMatchers {
```